server/utils/getFromDisk$.js

```diff
@@ -21,6 +22,12 @@ export default function getFromDisk$(challenge) {
       return _challenge;
     })
     .map(challenge => {
+      challenge.checksum = adler32.sum(
+        Buffer(JSON.stringify(challenge.title) +
+          JSON.stringify(challenge.description) +
```

Comment:
I figure that if we don't populate it with something that it will be `undefined` when it is used.  I've included it for testing purposes and validation.